homeassistant/components/lock/services.yaml

```diff
@@ -55,3 +55,59 @@ unlock:
     code:
       description: An optional code to unlock the lock with
       example: 1234
+
+set_lock_vacation_mode:
```

Comment:
If you add custom services for your platform, prefix them with your platform, `wink_set_lock_vacation_mode`